### ISSUE TITLE
Cannot select text

### ISSUE THREAD

**Issue by DeusFigendi** [jhass/insporation#2]
I cannot select and copy text from the stream view.
This is pretty important for quoting but also to copy text into other apps.

At least pressing long time the text doesn't start the selection state as android usually does.

Also I cannot copy nor share hyperlinks but maybe that is another issue.

**Comment by jhass** [comment]
Unfortunately not possible right now due to https://github.com/Sub6Resources/flutter_html/issues/169 which in turn is blocked by https://github.com/flutter/flutter/issues/43494

**Comment by Ravenbird** [comment]
I hope this will be fixed in the near future. By the way, the problem also extends to the download of pictures. At least I haven't found a possibility for that yet.

**Comment by Pafzedog** [comment]
Any news on this ?
It is not clear to me in the two links given above whether the problem is solved or not in upstream.
This is really a big obstacle to daily use.

**Comment by jhass** [comment]
The flutter side of things seems to have improved the status quo on this but on flutter_html nobody seems to be working on it.
